perceval/rendering/pdisplay.py

```diff
@@ -219,12 +226,89 @@ def pdisplay_state_distrib(sv: Union[StateVector, ProbabilityDistribution, BSCou
     s_states = tabulate(d, headers=headers, tablefmt=_TABULATE_FMT_MAPPING[output_format])
     return s_states
 
+    # labels on x- and y- axes
+
+
+def _generate_pauli_captions(nqubit):
+    from perceval.algorithm.tomography.tomography_utils import _generate_pauli_index
+    pauli_indices = _generate_pauli_index(nqubit)
+    pauli_names = []
+    for subset in pauli_indices:
+        pauli_names.append([member.name for member in subset])
+
+    basis = []
+    for val in pauli_names:
+        basis.append(''.join(val))
+    return basis
+
+
+def _get_sub_figure(ax, array, basis_name):
+    # Data
+    size = array.itemsize * 2
```

Comment:
`array.itemsize` is not the size or shape of the 2D matrix called "array". 
(For definition See fig or doc for more details: [https://docs.python.org/fr/3.10/library/array.html])

<img width="749" alt="Screenshot 2023-10-31 at 11 54 29" src="https://github.com/Quandela/Perceval/assets/37671249/691a67be-3ad0-4339-bc8d-87a79c7957ea">


My guess is that you want the size of the 2D matrix :

For example, a 2-qubit gate, like CNOT, chi would always be a 16x16 matrix). To retrieve this, you need (works on numpy arrays, even numpy.ndarray)-

`array.shape`

output exapmles: (16x16) for 2 qubit gate, (4x4) for 1 qubit gate, (64x64) for 3 qubit gate

Doing `array.itemsize` will give you 8 always for all the three qubit cases above. I think the error did not appear as you were in 2 qubit case.

Reply:
Fixed it thanks !